### DISCUSSION TITLE
Visualize avalon fingerprint?

### DISCUSSION THREAD

**Discussion by proexcuse** [rdkit/rdkit#5495] [Q&A]
Hi,

Is there any way to visualize avalon fingerprint?
It would be nice if I can use a method which receives avalon fingerprint bit and draws the corresponding substructure.

**Comment by greglandrum** [comment]
Hi @proexcuse, there's not currently any way to do this. The bits in the avalon fingerprint, which are described here https://pubs.acs.org/doi/10.1021/ci050413p, don't currently have an easy way to map them back to the atoms involved.

**Reply by proexcuse** [reply]
Thanks.
